src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <div className="App">
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      <p className="Person__age">
+        {person.age
```

Comment:
Instead of rendering 'null', you can just omit the entire 'Person__age' paragraph when there is no age provided. Use conditional rendering with '&&' operator 👍